src/middleware/and.js

```diff
@@ -0,0 +1,3 @@
+export default function And() {
+  this.root = this.instance
```

Comment:
This should be a method inside Test class because it doesn't fit the definition of a middleware.